projects/client/RabbitMQ.Client/src/client/api/AmqpTcpEndpoint.cs

```diff
@@ -138,6 +140,11 @@ public AmqpTcpEndpoint CloneWithHostname(string hostname)
             return new AmqpTcpEndpoint(hostname, _port, Ssl);
         }
 
+        /// <summary>
+        /// The default Amqp SSL protocols.
```

Comment:
How about "TLS protocol versions that will be used by default"?

Reply:
LGTM